static/copy.js

```diff
@@ -0,0 +1,14 @@
+let copyButton = document.getElementById('copy-button')
+
+function handleCopy() {
+  let copyLink = document.getElementById('url-link')
+  copyLink.value = window.location.href + copyLink.value;
+  copyLink.select()
+  copyLink.setSelectionRange(0, 99999); /* For mobile devices */
+
+  document.execCommand("copy");
+
+  alert("Copied the link: " + copyLink.value);
```

Comment:
And just something small, we noticed that the copy link is duplicated if you press it multiple times.

Reply:
Ooh, thanks for spotting this!